nucypher/cli/painting/staking.py

```diff
@@ -267,3 +274,50 @@ def paint_min_rate(emitter, staker):
     ~ Previously set ....... {prettify_eth_amount(raw_minimum)}
     ~ Effective ............ {prettify_eth_amount(minimum)}"""
     emitter.echo(rate_payload)
+
+
+def paint_staking_rewards(stakeholder, blockchain, emitter, num_past_periods, staking_address, staking_agent):
+    if num_past_periods:
+        economics = stakeholder.staker.economics
+        seconds_per_period = economics.seconds_per_period
+        days_per_period = economics.hours_per_period // 24
```

Comment:
Consider moving `days_per_period` into a constant or attribute on `economics`, it may be useful elsewhere in the codebase or tests.